### ISSUE TITLE
Necromancers should cast a discern-capped RoC before safe-room executes

### ISSUE THREAD

**Issue by cynesis** [rpherbig/dr-scripts#1261]
Until such time that Simutronics alters RoC/RoG+Liturgy to no longer allow Empaths to heal the Necromancer in question, a discern-capped RoC or RoG+Liturgy (RoC will likely be easier) should be buffed before executing safe-room. This ensures that a Necromancer can be healed.

I know this works for PC Empaths (safe_room_id and safe_room_empath). I have not tested it against NPC Empaths.

**Comment by izawwlgood** [comment]
Late to this, but, be aware that Necros get to a point where even with RoC or RoG+Liturgy up they cant be healed by empaths.

**Comment by MahtraDR** [comment]
@cynesis is this still a thing?

**Comment by izawwlgood** [comment]
No, and this can be solved by just setting up a before/after, which is what
people do.

**Comment by MahtraDR** [comment]
@rpherbig Can close. :)
